app/src/main/java/org/fossasia/openevent/app/OrgaApplication.java

```diff
@@ -94,7 +94,7 @@ public void onCreate() {
             StrictMode.setVmPolicy(
                 new StrictMode.VmPolicy.Builder()
                     .detectAll()
-                    .penaltyDeath()
+                    .penaltyLog()
```

Comment:
penaltyDialog

Reply:
There is no `penaltyDialog` for VmPolicy.

Reply:
OK